src/vdaf/prio3.rs

```diff
@@ -564,6 +573,48 @@ pub struct Prio3PrepareStep<F, const L: usize> {
     state: PrepareStep<F, L>,
 }
 
+impl<F: FieldElement, const L: usize> Encode for Prio3PrepareStep<F, L> {
+    fn encode(&self, bytes: &mut Vec<u8>) {
+        if let PrepareStep::Ready(..) = self.state {
+            panic!("encooding of ready state is not implemented");
```

Comment:
This wording leaves open the possibility that we will implement the encoding of the ready state in the future. Let's be more definite about it and provide some more context to help PPM implementers who will encounter this panic string while developing and may need some more help.

```suggestion
            panic!("encoding of ready state is not permitted. See Prio3PrepareStep documentation for details.");
```

Reply:
What about just adding a comment above the `panic!()`?

Reply:
Done, though worded slightly different